File: python/66 SumPositive.py
sum=0
while True:
    num=int(input("Enter a number: "))
    if num<0:
        continue
    else:
        sum+=num
        print(f"Sum is {sum}")
    exit=None
    exit=input("Type yes to exit or Press Enter to continue: ")
    if exit=="yes":
        break
    else:
        continue
print("END")

File: java/66 SumPositive.java
package code;
import java.util.*;
public class Test {

	public static void main(String[] args) {
		Scanner sc=new Scanner(System.in);
		int sum=0;
		String exit = null;
		while(true) {
			System.out.println("Enter a number: ");
			int input=sc.nextInt();
			sum+=input;
			System.out.println("Sum is "+sum);
			System.out.println("Type yes to exit or Type no to continue: ");
			exit=sc.next();
			if(exit.equalsIgnoreCase("yes")) {
				break;
			}
			else {
				continue;
			}
		}
		System.out.println("END");
	}
}